app/assets/stylesheets/application.css.scss.erb

```diff
@@ -24,6 +24,7 @@
  */
 
 /* BioPortal */
+@import "variables_stage";
```

Comment:
why do you call it `variables_stage` ?

Reply:
It contains the colors of stage (the light blue)

Reply:
and how you will do for agroportal ?

Reply:
Fixed